File: Modul 3/C/PRAK304-2310817220025-FirdaKhoirunisa.c
#include <stdio.h>

int main(){
    int bil;
    printf("Input\n");
    scanf("%d", &bil);

    printf("\nOutput\n");
    if(bil == 0){
        printf("Nol");
    }if(bil > 0 && bil <= 10){
        printf("Satuan");
    }if(bil > 10 && bil < 20){
        printf("Belasan");
    }if(bil >= 20 && bil < 100){
        printf("Puluhan");
    }if(bil >= 100){
        printf("Anda Menginput Melebihi Limit Bilangan");
    }
    return 0;
}

File: Modul 3/Python/PRAK304-2310817220025-FirdaKhoirunisa.py
print("Input")
bilangan = int(input())

print("\nOutput")
if(bilangan > 0 and bilangan <= 10):{
    print("Satuan")
}
elif(bilangan == 0):{
    print("Nol")
}
elif(bilangan > 10 and bilangan < 20):{
    print("Belasan")
}
elif(bilangan >= 20 and bilangan < 100):{
    print("Puluhan")
}
elif(bilangan >= 100):{
    print("Anda Menginput Melebihi Limit Bilangan")
}